OrbitQt/servicedeploymanager.cpp

```diff
@@ -335,6 +336,25 @@ outcome::result<void> ServiceDeployManager::ConnectToServer() {
   return outcome::success();
 }
 
+void ServiceDeployManager::StartWatchdog(std::string_view start_passphrase,
+                                         int interval_in_ms) {
+  orbit_service_task_->Write(start_passphrase);
+
+  const auto watchdog_send_timer = QPointer(new QTimer());
+
+  QObject::connect(watchdog_send_timer, &QTimer::timeout,
```

Comment:
Can you use the 4 argument overload of QObject::connect?

Reply:
Yes, I was wondering what the difference is?

Reply:
Ah sorry. You're right. This overload is fine. Just leave it like so.